### PR TITLE
Global word updates

### PR DESCRIPTION
**Pull request by danieltprice**
Auto-suspend -> "Autosuspend" when referred to as a feature, "autosuspend" otherwise 
Autoscaling -> "Autoscaling" when referred to as a feature. "autoscaling" otherwise 
Added Word List to Notion:
https://www.notion.so/neondatabase/Word-list-12ca0f536fcb47bd88b09d9ab89ba2ce?pvs=4

### CODE REVIEW

File: content/changelog/2023-05-29-console.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Neon supports up to 10,000 simultaneous connections when using [connection pooli
...
+The Postgres `max_connections` setting is now set according to your compute size. Previously, the `max_connections` setting was 100. The formula that calculates the `max_connections` setting is `RAM in bytes / 9531392 bytes`. For example, a compute with 12 GB of RAM has a `max_connections` setting of 1351. You can check the `max_connections` setting by running `SHOW max_connections;` from the Neon SQL Editor or from a client connected to Neon. When [autoscaling](/docs/introduction/autoscaling) is enabled, `max_connections` is calculated based on the minimum compute size in your autoscaling configuration.
```
Comment (bgrenon) [inline content/changelog/2023-05-29-console.md:7]:
```suggestion
The Postgres `max_connections` setting is now set according to your compute size. Previously, the `max_connections` setting was 100. The formula that calculates the `max_connections` setting is `RAM in bytes / 9531392 bytes`. For example, a compute with 12 GB of RAM has a `max_connections` setting of 1351. You can check the `max_connections` setting by running `SHOW max_connections;` from the Neon SQL Editor or from a client connected to Neon. When [Autoscaling](/docs/introduction/autoscaling) is enabled, `max_connections` is calculated based on the minimum compute size in your autoscaling configuration.
```

Comment (bgrenon) [reply content/changelog/2023-05-29-console.md:7]: Would this one be considered a feature, hence capitalized?

File: content/docs/guides/auto-suspend-guide.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+Neon's [Autosuspend](/docs/introduction/auto-suspend) feature controls when a Neon compute instance transitions to an `Idle` state (scales to zero) due to inactivity. This guide demonstrates how to configure the autosuspend setting in your Neon project.
 
-[Neon Pro Plan](/docs/introduction/pro-plan) users can configure the _Auto-suspend_ setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the _Auto-suspend_ setting for a new or existing project sets the project's default Auto-suspend setting, which is used from that point forward when creating new compute endpoints.
+[Neon Pro Plan](/docs/introduction/pro-plan) users can configure the autosuspend setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the Autosuspend setting for a new or existing project sets the project's default autosuspend setting, which is used from that point forward when creating new compute endpoints.
```
Comment (bgrenon) [inline content/docs/guides/auto-suspend-guide.md:11]:
```suggestion
[Neon Pro Plan](/docs/introduction/pro-plan) users can configure the autosuspend setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the autosuspend setting for a new or existing project sets the project's default autosuspend setting, which is used from that point forward when creating new compute endpoints.
```

Comment (bgrenon) [reply content/docs/guides/auto-suspend-guide.md:11]: I think this 2n should be lowercase as well - but disregard if I've got it wrong

File: content/docs/guides/guides-intro.md [hunk 3]
```diff
@@ -91,21 +91,21 @@ Automatically scale compute resources up and down based on demand.
...
 
-### Auto-suspend
+### Autosuspend
 
 Control when Neon compute resources scale to zero.
 
 <DetailIconCards>
 
-<a href="/docs/introduction/auto-suspend" description="Discover how Neon can reduce your compute to zero when not in use" icon="hourglass">Learn about Auto-suspend</a>
+<a href="/docs/introduction/auto-suspend" description="Discover how Neon can reduce your compute to zero when not in use" icon="hourglass">Learn about Autosuspend</a>
```
Comment (bgrenon) [inline content/docs/guides/guides-intro.md:106]: Should the titles be lowercase or uppercase? They are not consistent between autoscaling card titles and Autosuspend card titles.

**Review by bgrenon — APPROVED** [review]

File: content/docs/guides/guides-intro.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/guides/guides-intro.md:106]: Fixed.

**Pull request merged**